valdr-ng/projects/valdr-ng/package.json

```diff
@@ -17,9 +17,9 @@
     "Stanoja"
   ],
   "peerDependencies": {
-    "@angular/forms": "^15.0.0",
-    "@angular/common": "^15.0.0",
-    "@angular/core": "^15.0.0"
+    "@angular/core": "^16.1.2",
```

Comment:
Shouldn't the peerDependency point to any angular 16 version? Otherwise we would technically not allow someone with angular 16.0.0 to use the library.. right?